app/src/main/java/ch/epfl/sdp/blindly/settings/Settings.kt

```diff
@@ -69,8 +69,7 @@ class Settings : AppCompatActivity() {
         val intent = Intent(this, SettingsShowMe::class.java).apply {
             putExtra(EXTRA_SHOW_ME, currentShowMe)
         }
-
-        //startActivityForResult(EXTRA_SHOW_ME).launch(intent)
+        //TODO fix me

```

Comment:
Is there something that doesn't work ?

Reply:
This one uses a deprecated function, I just added a //TODO when I came across this